packages/edit-site/src/components/block-editor/index.js

```diff
@@ -157,31 +131,6 @@ export default function BlockEditor( { setIsInserterOpen } ) {
 	const isTemplatePart = templateType === 'wp_template_part';
 	const hasBlocks = blocks.length !== 0;
 
-	const NavMenuSidebarToggle = () => (
```

Comment:
You're removing logic from edit-site here (site editor) without replacing it, is that intended?

Reply:
Yeah. The button is now created in the edit-post plugin and appears in both places. The old button opened the navigation sidebar. We don't want that at the moment as are experimenting with adding a list view in the navigation  block's inspector controls.

Reply:
I was surprised too!

Reply:
@scruffian Did we track this down?

Reply:
yes it's fixed.